pkg/service/generator.go

```diff
@@ -497,6 +538,25 @@ func generateConfigMapForModules(emqx v1beta1.Emqx, sts *appsv1.StatefulSet) (*c
 	return cm, sts
 }
 
+func generateConfigMapForTelegraf(emqx v1beta1.Emqx, sts *appsv1.StatefulSet) (*corev1.ConfigMap, *appsv1.StatefulSet) {
```

Comment:
This function doesn't seem to have a useful `StatefulSet` argument, meybe we should merge `generateVolumeForTelegraf` in this